webapp/src/index.js

```diff
@@ -1,21 +1,44 @@
-import React from 'react'
-import {render} from 'react-dom'
-import {Provider} from 'react-redux'
-import configureStore from './store/configureStore'
-import Page from './ui/page'
+import React from 'react';
+import {render} from 'react-dom';
+import {Provider} from 'react-redux';
+import configureStore from './store/configure-store';
+import Page from './ui/page';
+import {Container} from '@material-ui/core';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+import {createMuiTheme, ThemeProvider} from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+// eslint-disable-next-line import/no-unassigned-import
 import 'typeface-roboto';
-import {
-  Container
-} from '@material-ui/core'
 
+const store = configureStore();
 
-const store = configureStore()
+const App = () => {
+	const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');
 
-render(
-  <Provider store={store}>
-    <Container>
-      <Page />
-    </Container>
-  </Provider>,
-  document.getElementById('root')
-)
+	const theme = React.useMemo(
+		() =>
+			createMuiTheme({
+				palette: {
+					type: prefersDarkMode ? 'dark' : 'light'
```

Comment:
After downloading your PR I wasn't able to actually get the dark theme to work. I looked around and found it's actually the browser settings that control this, correct?

Can we make dark mode the default and then add a toggle for the people that want light mode?

Reply:
Correct, it's currently controlled by the OS setting (works on macOS and Windows in Chrome and Firefox, not sure about Linux desktop environments).

How about we make the default the system setting, and then the user can change it if they want with a toggle?

Reply:
Works for me, as long as there's some way to change the theme alongside OS themes.

Reply:
Check out 30fccb5. Works well on desktop, but kinda awkward on mobile:
![Screen Shot 2020-04-29 at 9 25 12 PM](https://user-images.githubusercontent.com/7410405/80665812-16aab080-8a60-11ea-9053-156cf7dee334.png)

Reply:
I'm gonna mark this as resolved for now. An easy way to fix mobile (and desktop for that matter) positioning of the theme button would be to add a navbar, but we don't need that yet.